pages/user guides/code_analysis_overview.md

```diff
@@ -42,9 +42,103 @@ It measures and determines several important aspects of an application code base
+```
+
+- Identified software dependencies package names, and versions, for example:
+```javascript
+{
+  "from_file": "/src/home/pnr/tidal/schoolbus/Server/Serilog.Sinks.SbiPostgreSql/Serilog.Sinks.SbiPostgreSql.csproj",
+  "language": "CSharp",
+  "name": ".NET Core",
+  "owner": null,
+  "version": "3.1"
+},
+{
+  "from_file": "/src/home/pnr/tidal/schoolbus/client/package-lock.json",
+  "language": "JavaScript",
+  "name": "@babel/code-frame",
+  "owner": null,
+  "version": "7.10.4"
+}
+```
+
+- Identified patterns and implementations in code, for example, the detection of filesystem usage in Python:
+```javascript
+{
+  "check_id": "filesystem-python",
+  "end": {
+    "col": 19,
+    "line": 132,
+    "offset": 3703
+  },
+  "extra": {
+    "engine_kind": "OSS",
+    "fingerprint": "f50c74db097532ad463e76e178545defc5ed47fbaf364527da876a75e33481708f97308217f2364b57a9986f939c65030a21853622be9380d16c08940a95cbb6_0",
+    "is_ignored": false,
+    "lines": "    if not os.path.exists(output_dir):",
+    "message": "Filesystem usage",
+    "metadata": {},
+    "metavars": {},
+    "severity": "INFO"
+  },
+  "path": "/src/home/pnr/tidal/schoolbus/ApiSpec/CCWJurisdiction/csv2json.py",
+  "start": {
+    "col": 12,
+    "line": 132,
+    "offset": 3696
+  }
+}
+```
 
-## Security
-The entire code analysis process takes place locally on your machine. It will scan your files locally looking for common patterns and information when modernizing. The only data that is captured and sent from the analysis are the results of the analysis and the associated metadata. You can view all of the raw analysis results that are sent to Tidal Accelerator before they are sent, by viewing the contents of the results file locally.
+These are snippets from a real [analysis results file](code-analysis-20231205-123731-all.json). To verify and confirm these findings, and see what the full results include on either your application or a sample application see the next section below.
+## Validate and confirm
+You can view all of the raw analysis results before they are sent or seen by anyone. you can perform the entire analysis locally without any network connectivity to ensure this.
+The only data that is captured and sent from the analysis are the results of the analysis and the associated metadata.
 
+To see an example of the information that is captured and sent:
+
+1. Download a copy of this open-source ['schoolbus' application](https://github.com/tidalmigrations/schoolbus) by [cloning it with `git`](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) or [downloading a zip file](https://docs.github.com/en/repositories/working-with-files/using-files/downloading-source-code-archives#downloading-source-code-archives-from-the-repository-view)(and then unzipping it) from GitHub. If you have source code locally that you would like to try it on we recommend you use that.
```

Comment:
This may be tough to balance but having a smaller repo to analyze might produce an easier to understand analysis result. 15k lines feels like a large JSON object to give to people to read.

Reply:
Yea good point. Can look to find simpler examples and replace this then.